api/models/Invitation.js

```diff
@@ -47,7 +47,7 @@ module.exports = bookshelf.Model.extend(Object.assign({
     const { transacting } = opts
     return Membership.where({
       user_id: userId, community_id: this.get('community_id')
-    }).fetch()
+    }).fetch({withRelated: 'community'})
```

Comment:
can you add a comment here about how this is so that the data is available for callers, even though it's not used in the code below?